odata-protocol/11.7 Batch Requests.md

```diff
@@ -140,11 +142,14 @@ identifier prefixed with a `$` character as the unquoted value of the
 
 ### ##subsubsec Referencing Values from Response Bodies
 
-Services MAY support using values from a response body in the query part
-of the URL or in the request body of subsequent requests. Value
-references consist of a `$` character, followed by the request
-identifier of the preceding request, and optionally followed by a valid
-OData path.
+Services MAY support using values from a response body in the query part of
+the URL or in the request body of subsequent requests. Value references
+consist of a `$` character, followed by the request identifier of the preceding
+request. They evaluate to the referenced value, that is the value represented
+by the response body of that preceding request. If that value is a collection
+or has an entity or complex type, the value reference MAY be followed by a
+forward slash and a path expression that is evaluated relative to the
```

Comment:
No forward slash if it is a collection of entities and follow it with a paren-style key.